packages/compass-export-to-language/package.json

```diff
@@ -54,6 +54,7 @@
     "reformat": "npm run prettier -- --write ."
   },
   "peerDependencies": {
+    "@mongodb-js/compass-components": "*",
```

Comment:
Star versions lead to weird behavior with npm@8, I cleaned it up awhile ago, but seems like this is getting introduced again. Can you clean these up please in this package while you are adding new dependencies here?

Reply:
Sure!

Reply:
If we find this happens often it may be worth adding something to our `depalign` script.